Sources/SentryCrash/Recording/Tools/SentryCrashUUIDConversion.h

```diff
@@ -15,7 +15,7 @@ extern "C" {
  * @param dst A buffer with the length of 37 to hold the human readable UUID.
  *
  */
-static void
+static inline void
```

Comment:
Was it a warning that led to adding this?
I wonder why explicilty mark this inline. It's often a compiler choice given the trade off in performance vs binary size

Reply:
Yes, this caused the following warning `Unused function 'sentrycrashdl_convertBinaryImageUUID'`. This is because this function is marked as static in a header file so each when you include this header file you get an own internal copy of the function. Which is basically what `inline` does. Thinking about it again it doesn't make sense to inline this function. So I changed it now and defined it as a normal c function. While doing this I discovered that the `*dst` was not null-terminated. I don't understand fully how it worked before. I added tests for it and also tested it on a simulator.